src/VerticalTabs/stories.tsx

```diff
@@ -38,7 +37,7 @@ const LoremTooltip = (
   </>
 );
 
-const VerticalTabsStories = storiesOf(`${__dirname}`, module).add('Basic', () => {
+storiesOf(`VerticalTabs`, module).add('Basic', () => {
   const [activeItem, setActiveItem] = React.useState(0);
```

Comment:
Not sure which line but Vertical Tabs break on localhost
`TypeError: Cannot read properties of undefined (reading 'Item')`

Reply:
could not recreate this one

Reply:
Will resolve this one offline (also occurring in Pipe) -- storybook is building and running so this is specific to the components